src/components/atoms/Text/Text.tsx

```diff
@@ -6,27 +6,48 @@ import { colors, TColors } from '../../../constants/colors';
 
 export interface ITextProps extends HTMLAttributes<HTMLSpanElement> {
   /**
-   * The weight of the rendered text. Avoid the use of `semibold` as for now.
+   * The weight of the rendered text.
+   * @default 'regular'
    */
-  weight?: keyof typeof typography.weights;
+  weight?: 'regular' | 'bold';
   /**
-   * The size you want to render your text at, currently only `14px` | `16px` and `20px` supported.
+   * The size you want to render your text at, currently only `13px` | `15px` and `18px` supported.
+   * @default 'body'
    */
-  size?: keyof typeof typography.sizes.text | 'lead';
+  size?: keyof typeof typography.sizes.text;
   /**
    * The HTML5 tag you want to render your text on, currently only `<span>` and `<p>` are supported.
+   * @default 'span'
    */
   as?: 'span' | 'p';
   /**
    * Whether to add some margin below the rendered text or not. Use it to give meaningful white-space.
+   * @default false
    */
   mb?: boolean;
   /**
-   * Accepts a subset of the Zopa brand colors. Same as the ones accepted by `<Heading />`.
+   * Whether to render the text in all caps or not.
+   * @default false
+   */
+  capitalize?: boolean;
+  /**
+   * Where the rendered text should be aligned to.
+   * @default 'left'
```

Comment:
Well, technically there is no default from what I see which works just fine. If you decide to add a default value, than it should be `inherit` rather than `left`

Reply:
same thing goes for `Heading`

Reply:
Yup I'll update 👍